mapie/conformity_scores.py

```diff
@@ -43,12 +46,13 @@ def __init__(
             by default sys.float_info.epsilon.
         """
         self.sym = sym
-        self.eps = eps
         self.consistency_check = consistency_check
+        self.compute_score_weights = compute_score_weights
+        self.eps = eps
 
     @abstractmethod
     def get_signed_conformity_scores(
-        self, y: ArrayLike, y_pred: ArrayLike,
+        self, y: ArrayLike, y_pred: ArrayLike, conformity_scores: ArrayLike
```

Comment:
I think this is a mistake: `conformity_scores` should not be an argument to `get_signed_conformity_scores`.